src/error.ts

```diff
@@ -670,7 +670,6 @@ function makeWriteConcernResultObject(input: any) {
 export class MongoWriteConcernError extends MongoServerError {
   /** The result document (provided if ok: 1) */
   result?: Document;
```

Comment:
Did you mean to remove `errInfo` from the class?

Reply:
Yes it's already defined in the parent, you'll notice a few other removals, if the override doesn't actually narrow the type then there's no purpose for it.